data/linux-bridge/002-linux-bridge.yaml

```diff
@@ -39,23 +39,23 @@ spec:
             - /bin/bash
             - -ce
             - |
-              echo 'Installing bridge and tuning CNIs'
-              cni_mount_dir=/opt/cni/bin
-              sourcebinpath=/usr/src/github.com/containernetworking/plugins/bin
-              cp --remove-destination ${sourcebinpath}/bridge ${cni_mount_dir}/cnv-bridge
-              cp --remove-destination ${sourcebinpath}/tuning ${cni_mount_dir}/cnv-tuning
+              if [ find ${cni_mount_dir}/bridge &>/dev/null ] && [ ! -f ${cni_mount_dir}/upstream ]; then
+                ln -s ${cni_mount_dir}/bridge ${cni_mount_dir}/cnv-bridge
+                ln -s ${cni_mount_dir}/tuning ${cni_mount_dir}/cnv-tuning
+              else
+                touch ${cni_mount_dir}/upstream
+                cni_mount_dir=/opt/cni/bin
+                sourcebinpath=/usr/src/github.com/containernetworking/plugins/bin
+                cp --remove-destination ${sourcebinpath}/bridge ${cni_mount_dir}/cnv-bridge
+                cp --remove-destination ${sourcebinpath}/tuning ${cni_mount_dir}/cnv-tuning
 
-              echo 'Checking bridge and tuning CNIs deployment on node'
-              printf -v bridgechecksum "%s" "$(<$sourcebinpath/bridge.checksum)"
-              printf -v tuningchecksum "%s" "$(<$sourcebinpath/tuning.checksum)"
-              printf "%s %s" "${bridgechecksum% *}" "${cni_mount_dir}/cnv-bridge" | sha256sum --check
-              printf "%s %s" "${tuningchecksum% *}" "${cni_mount_dir}/cnv-tuning" | sha256sum --check
-
-              # Some projects (e.g. openshift/console) use cnv- prefix to distinguish between
-              # binaries shipped by OpenShift and those shipped by KubeVirt (D/S matters).
-              # Following two lines make sure we will provide both names when needed.
-              find ${cni_mount_dir}/bridge &>/dev/null || ln -s ${cni_mount_dir}/cnv-bridge ${cni_mount_dir}/bridge
-              find ${cni_mount_dir}/tuning &>/dev/null || ln -s ${cni_mount_dir}/cnv-tuning ${cni_mount_dir}/tuning
+                printf -v bridgechecksum "%s" "$(<$sourcebinpath/bridge.checksum)"
+                printf -v tuningchecksum "%s" "$(<$sourcebinpath/tuning.checksum)"
+                printf "%s %s" "${bridgechecksum% *}" "${cni_mount_dir}/cnv-bridge" | sha256sum --check
+                printf "%s %s" "${tuningchecksum% *}" "${cni_mount_dir}/cnv-tuning" | sha256sum --check
+                find ${cni_mount_dir}/bridge &>/dev/null || ln -s ${cni_mount_dir}/cnv-bridge ${cni_mount_dir}/bridge
+                find ${cni_mount_dir}/bridge &>/dev/null || ln -s ${cni_mount_dir}/cnv-tuning ${cni_mount_dir}/tuning
+              fi
```

Comment:
in case of redeployment, wouldn't this keep the first bridge that was created ?
we can use 'ln -sf` for upstream if desired